app/web/src/components/PageTitle.tsx

```diff
@@ -8,8 +8,19 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function PageTitle(props: TypographyProps) {
+type PageTitleProps = TypographyProps & {
+  className?: string;
+};
+
+export default function PageTitle({ className, ...props }: PageTitleProps) {
```

Comment:
`className?` should already be in `TypographyProps` so you don't actually need the new type.

Reply:
Changed it in last commit